### ISSUE TITLE
PowerShell 7.2.0 unable to launch

### ISSUE THREAD

**Issue by ZiaUrR3hman** [PowerShell/PSReadLine#2996]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
### Exception

System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot

![image](https://user-images.githubusercontent.com/9167406/141072033-2d34deb4-5d52-4f90-8ba1-092ee943a23b.png)


### Environment data

```console
### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.19043
BufferWidth: 118
BufferHeight: 27

Last 0 Keys


```
```


### Steps to reproduce

after installing new version on my system, I got the issue..

### Expected behavior

it should launch the powershell

### Actual behavior

unable to type anything on powershell

**Comment by bulax** [comment]
Please Update PSreadline to version 2.2.0-beta4
This will fix problem

**Comment by cosmo0** [comment]
> Please Update PSreadline to version 2.2.0-beta4 This will fix problem

In case like me you're wondering how, follow the installation steps here: https://github.com/PowerShell/PSReadLine

**Comment by abdullahiabdirahman** [comment]
Facing same issue. 

> Please Update PSreadline to version 2.2.0-beta4 This will fix problem

mine is updated to this version and still throws same exception

**Comment by abdullahiphys** [comment]
> Please Update PSreadline to version 2.2.0-beta4 This will fix problem

yes this fixed the issue. Thanks @bulax

**Comment by mdymel** [comment]
I have tried upgrading with  PSReadLine with: 
`powershell -noprofile -command "Update-Module PSReadLine -Force -AllowPrerelease -Verbose"`
But it still fails to open Powershell and shows: 
```
### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 120
BufferHeight: 30
```

**Comment by iSazonov** [comment]
@mdymel We should close all pwsh and run `pwsh -noprofile -command "Update-Module PSReadLine -Force -AllowPrerelease -Verbose"`

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
